enterprise/backup/src/test/java/org/neo4j/backup/BackupServiceIT.java

```diff
@@ -673,13 +678,14 @@ private void checkLastCommittedTxIdInLogAndNeoStore( long txId ) throws IOExcept
         }
 
         // Assert last committed transaction is correct in neostore
-        NeoStoreUtil store = new NeoStoreUtil( backupDir, fileSystem );
+        NeoStoreUtil store = new NeoStoreUtil( backupDir, pageCacheRule.getPageCache( fileSystem ) );
         assertEquals( txId, store.getLastCommittedTx() );
     }
 
     private long getLastTxChecksum()
     {
-        return new NeoStoreUtil( backupDir ).getValue( Position.LAST_TRANSACTION_CHECKSUM );
+        return new NeoStoreUtil( backupDir, pageCacheRule.getPageCache( fileSystem ) ).getValue( Position
+                .LAST_TRANSACTION_CHECKSUM );
```

Comment:
More odd formatting. Also note that every time you call `PageCacheRule.getPageCache()` you get a new one back. You might want to get it once per test, and then pass it around.